quiche/src/ffi.rs

```diff
@@ -845,7 +852,14 @@ pub extern fn quiche_conn_stream_send(
     match conn.stream_send(stream_id, buf, fin) {
         Ok(v) => v as ssize_t,
 
-        Err(e) => e.to_c(),
+        Err(e) => {
+            match e {
+                Error::StreamReset(error) => unsafe { *application_error = error },
```

Comment:
```suggestion
                Error::StreamReset(error) => *out_error_code = error,
```

Reply:
Renaming it, but cannot drop unsafe - it's dereferencing a raw pointer.

Reply:
ICYMI, the suggestion is also to change `*mut` to `&mut` which should allow removing the `unsafe`.

Reply:
Oh, you're right. I missed that. Changing...